docs/admin/reference/hardware.rst

```diff
@@ -52,118 +60,25 @@ The 2nd-generation ThinkPad T14 **with an 11th-generation Intel Core processor**
-  A Tails USB can be used for the verification and conversion process described below, but the Lenovo support site blocks requests over Tor, preventing the ISO download. To work around this, either:
-
-  - download the BIOS ISO on a different computer and transfer it to Tails using a USB stick, or
-  - download the ISO in Tails using the Unsafe Browser as follows:
-
-    - Start Tails with an administration password set and the Unsafe Browser enabled under "Additional Settings" on the Welcome Screen.
-    - Open the Unsafe Browser: **Applications > Internet > Unsafe Browser** and find and download the ISO
-    - Note the filename, as you'll need it for subsequent steps.
-    - Leave the Unsafe Browser running, and open a terminal via **Applications > System Tools > Terminal**.
-    - Copy the ISO to the desktop with the command:
-
-      .. code-block:: sh
-
-        sudo cp /var/lib/unsafe-browser/chroot/home/clearnet/Downloads/<fileName.iso> ~amnesia/Desktop
+After Qubes starts for the first time, when ``sys-net`` fails to start, follow the troubleshooting instructions for :ref:`reset_pci`, but only for the ``dom0:00_1f.6`` Ethernet device.
 
-    - Fix the ISO file's ownership with the command:
+.. _framework_13_series:
 
-      .. code-block:: sh
+Framework 13 13th generation laptop
+***********************************
+TK text and description here
 
-        sudo chown amnesia:amnesia ~amnesia/Desktop/<fileName.iso>
-
-- Verify the checksum of the downloaded ISO file using the following command, comparing it against the checksum in the file listing above:
-
-  .. code-block:: sh
-
-    sha256sum /path/to/downloaded.iso
-
-- Create a USB-bootable version of the ISO using the command:
-
-  .. code-block:: sh
-
-    geteltorito <path/to/CDISO> > usb-bios.iso
-
-  .. note:: To install the ``geleltorito`` utility on Debian-based systems, use the command
-
-    .. code-block:: sh
-
-      sudo apt install genisoimage
-
-    To install it on Fedora-based systems, use the command:
-
-    .. code-block:: sh
-
-      sudo dnf install geteltorito genisoimage
-
-- Plug in a USB and check its device name with the ``lsblk`` command - use the root device name below, not a partition (eg. ``/dev/sdc`` instead of ``/dev/sdc1``).
-
-- Write the BIOS update ISO to the USB using the following command:
-
-  .. code-block:: sh
-
-    sudo dd if=usb-bios.iso of=/dev/sdX bs=1M && sync
-
-  where ``sdX`` is the device name verified above.
-
-  .. caution::
-
-    The ``dd`` command will wipe data on the targeted device. Make sure that you use the correct device name.
-
-  Once complete, remove the USB.
-
-- Plug the USB into the ThinkPad.
-
-- Boot the ThinkPad and follow the prompts to enter its startup and boot menus, likely via the <Enter> and <F12> keys, respectively.
-
-- Follow the on-screen instructions to update the BIOS, including any mandatory reboots. Note that the instructions may refer to an update CD instead of your update USB.
-
-USB-C ports
```

Comment:
Was the decision to remove the section on USB-C ports because this information is no longer applicable, or because it's too in-the-weeds?

Reply:
Bit of both - it references 8th-gen T480s so n/a, and if it is caused by bios settings in install section it can probably be a note there.